applications/MeshMovingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -53,6 +55,23 @@ void AddCustomUtilitiesToPython(pybind11::module& m) {
         .def("ProjectVirtualValues3D", &ExplicitFixedMeshALEUtilities::ProjectVirtualValues<3>)
         .def("UndoMeshMovement", &ExplicitFixedMeshALEUtilities::UndoMeshMovement);
 
+    py::class_<LinearTransform, LinearTransform::Pointer>(m, "LinearTransform")
+        .def(py::init<const array_1d<double,3>&, const double, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def(py::init<const array_1d<double,3>&, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def(py::init<const Quaternion<double>&, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const array_1d<double,3>&, const double, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const array_1d<double,3>&, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const Quaternion<double>&, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
+        .def("SetTranslation", &LinearTransform::SetTranslation)
+        .def("Apply", &LinearTransform::Apply)
+        ;
+
+    py::class_<ParametricLinearTransform, ParametricLinearTransform::Pointer>(m, "ParametricLinearTransform")
+        .def(py::init<const Parameters&, const Parameters&, const Parameters&, const Parameters&>())
+        .def(py::init<const Parameters&, const Parameters&, const Parameters&>())
```

Comment:
```suggestion
        .def(py::init<const Parameters, const Parameters, const Parameters, const Parameters>())
        .def(py::init<const Parameters, const Parameters, const Parameters>())
```

`Kratos::Parametes` should be passed by value due to internal memory management